File: Tests/CK.TypeScript.Tests/CrisLike/Model/ICommandAuthNormal.cs
using CK.Core;

#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved

namespace CK.CrisLike;

/// <summary>
/// Extends <see cref="ICommandAuthUnsafe"/> to ensure that the authentication level is <see cref="AuthLevel.Normal"/>
/// (or <see cref="AuthLevel.Critical"/>).
/// </summary>
[CKTypeDefiner]
public interface ICommandAuthNormal : ICommandAuthUnsafe
{
}

File: Tests/CK.TypeScript.Tests/TSInlineTests/CrisLike_commands_and_results/ck-gen/CK/CrisLike/ICommandAuthNormal.ts
import { ICommandAuthUnsafe } from './ICommandAuthUnsafe';

/**
 * Extends ICommandAuthUnsafe to ensure that the authentication level is 
 * ~~!:AuthLevel.Normal~~ (or 
 * ~~!:AuthLevel.Critical~~).
 **/
export interface ICommandAuthNormal extends ICommandAuthUnsafe {
readonly _brand: ICommandAuthUnsafe["_brand"] & {"28":any};
}
